src/Data/LCA/Online/Naive.hs

```diff
@@ -142,6 +140,10 @@ _                ~= _                = False
 {-# INLINE (~=) #-}
 
 -- | /O(h)/ Compute the lowest common ancestor of two paths
+--
+-- >>> let fromList' = fromList . map (flip (,) ())
```

Comment:
Personally, I find `\x -> (x, ())` easier to read than `flip (,) ()`, but I don't feel strongly about this.

Reply:
It's the same example as in other modules. I simply copied that. I could change all.

Reply:
Ah, I overlooked the fact that this is defined elsewhere in the library. In that case, I agree that it's best to just copy it for the sake of this PR.